pkg/cri/server/container_create_windows_test.go

```diff
@@ -248,3 +253,128 @@ func TestHostProcessRequirements(t *testing.T) {
 		})
 	}
 }
+
+func isEqualStringArrays(values, expected []string) bool {
+	if len(values) != len(expected) {
+		return false
+	}
+
+	for i, x := range expected {
+		if values[i] != x {
+			return false
+		}
+	}
+	return true
+}
+
+func assertEqualsStringArrays(values, expected []string) error {
+	if !isEqualStringArrays(values, expected) {
+		return fmt.Errorf("expected %s, but found %s", expected, values)
+	}
+	return nil
+}
```

Comment:
This isn't needed, just use `assert.Equal`:

```go
import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestEquals(t *testing.T) {
	greeting := []string{"hi", "hello"}
	greeting2 := []string{"hi", "hello"}
	assert.Equal(t, greeting, greeting2)
}

func TestDiff(t *testing.T) {
	greeting := []string{"hi", "hello"}
	greeting2 := []string{"hi", "howdy"}
	assert.Equal(t, greeting, greeting2)
}
```

```shell
testassert go test -v ./howdy
=== RUN   TestEquals
--- PASS: TestEquals (0.00s)
=== RUN   TestDiff
    howdy_test.go:18: 
                Error Trace:    /Users/dcantah/go/src/github.com/dcantah/howdy/howdy_test.go:18
                Error:          Not equal: 
                                expected: []string{"hi", "hello"}
                                actual  : []string{"hi", "howdy"}
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -2,3 +2,3 @@
                                  (string) (len=2) "hi",
                                - (string) (len=5) "hello"
                                + (string) (len=5) "howdy"
                                 }
                Test:           TestDiff
--- FAIL: TestDiff (0.00s)
FAIL
```

Reply:
You were calling fatal after these so possibly use `require.Equal` instead as that will end the test just like t.Fatal would